tests/dflowfm/test_tim.py

```diff
@@ -78,6 +78,28 @@ def test_serialize_data(self, input_data, reference_path):
 
 
 class TestTimModel:
+    triple_data_for_timeseries_floats = [
+        TimRecord(time=10, data=[1.232, 2.343, 3.454]),
+        TimRecord(time=20, data=[4.565, 5.676, 6.787]),
+        TimRecord(time=30, data=[1.5, 2.6, 3.7]),
+    ]
+
+    single_data_for_timeseries_floats = [
+        TimRecord(time=0.0000000, data=[0.0000000]),
```

Comment:
Do you really need this many entries? Can't we shorten the testdata to make it more readable? (And to make the file smaller)

Reply:
this is the example I got when I started on the tim model